### DISCUSSION TITLE
see compilation error in kernel building

### DISCUSSION THREAD

**Discussion by MattiaGrigoli** [Joshua-Riek/ubuntu-rockchip#1138] [Q&A]
is there a way to see the compilation error while compiling the kernel?
i'm building a custom kernel and when there's an error the only thing i can see is a generic "make error 2"
the command i use is the one for building the whole image so it's using debian/rules for compiling

**Comment by hakotani-o** [comment]
I also had trouble figuring out the cause of make error 2.
At that time,

fakeroot debian/rules binary-headers binary-rockchip do_mainline_build=true 2>../err

I remember trying to write the error output to a file.

Thank you.
